vortex-flatbuffers/src/lib.rs

```diff
@@ -55,6 +57,7 @@ pub mod dtype;
 #[allow(clippy::unwrap_used)]
 #[allow(clippy::borrow_as_ptr)]
 #[allow(dead_code)]
+#[allow(mismatched_lifetime_syntaxes)]
```

Comment:
Even after updating to latest flatc, it still generated bad code, so I just ignore it.